src/web/components/StickyAd.tsx

```diff
@@ -4,14 +4,29 @@ import { css } from 'emotion';
 import { AdSlot } from '@root/src/web/components/AdSlot';
 import { namedAdSlotParameters } from '@root/src/model/advertisement';
 
+// Note that 'StickyAd' seems heavily scoped
+// to the right column, if we want to 'stickyAd' somewhere
+// else then we'll need to send props through
+
 const adSlotWrapper = css`
     position: static;
-    height: 1059px;
 `;
 
 const stickyAdSlot = css`
-    position: sticky;
-    top: 0;
+    > .ad-slot__content {
```

Comment:
`.ad-slot__content` only exists when an ad is inserted, so we don't have a forced height with no ads